src/Components/Modal/Modal.spec.tsx

```diff
@@ -3,20 +3,23 @@ import * as React from "react";
 import Modal from "./Modal";
 import { Text } from "react-native";
 
-it("should not render modal if isVisible prop equal false", async () => {
+jest.useFakeTimers();
```

Comment:
You do not use any timers in your application's code. Why do you need to use fake timers in the test?